src/openms/source/ANALYSIS/TOPDOWN/sources.cmake

```diff
@@ -9,6 +9,7 @@ FLASHDeconvHelperStructs.cpp
 MassFeatureTrace.cpp
 PeakGroup.cpp
 QScore.cpp
+Qvalue.cpp
```

Comment:
Maybe QScore to be consistent?

Reply:
Qvalue and Qscore are two things... Qscore  - raw quality score. Qvalue - q value from QScore. :)

Reply:
Yep sorry there was a mistake in my initial comment. I was suggesting to use uppercase V for Value in the file and classname.

Reply:
Just because QScore and QValue, you know?

Reply:
Aha.. I see. I think I need to change the name QScore to Qscore. Thanks!!